### PR TITLE
Add Group Controller

### PR DESCRIPTION
**Pull request by jestradadeveloper**
# Description
This commit create the routes and the controller with its actions for Group model in order to handle its requests from routing. Also added request and controllers tests. Add annotate gem for helping us to view the models attributes.

# Controller  Added
- GroupController
# Route  Added
- resources :groups
# Actions added
-index
-new
-show
-create
-edit
-update
-destroy

# Private methods added to GroupController 
-set_group
-group_params

### CODE REVIEW

File: app/controllers/groups_controller.rb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        redirect_to group_path, notice:  "El grupo ha sido eliminado exitosamente"
+    end
+    private
+        def set_group
+            @group = Group.find params[:id]
+        end
+        def group_params 
+            params.require(:group).permit(:name,:description,:amount)
+        end
+end
```
Comment (JuanequeX) [inline app/controllers/groups_controller.rb:40]: Check your indentation in this file and add a blank space in every method

**Review by JuanequeX — CHANGES_REQUESTED** [review]

File: app/controllers/groups_controller.rb (hunk 1 above)
Comment (jestradadeveloper) [reply app/controllers/groups_controller.rb]: I already made this change. Thanks for the CR

**Review by JuanequeX — DISMISSED** [review]
LGMT

**Review by CUBOFIG — APPROVED** [review]
LGTM!

**Review by JuanequeX — APPROVED** [review]
LGTM

**Pull request merged**